aiidalab_widgets_base/viewers.py

```diff
@@ -233,7 +234,7 @@ def atoms_in_representaion(self, structure: ase.Atoms | None = None):
         if structure:
             if self.uuid in structure.arrays:
                 return structure.arrays[self.uuid] >= self.atom_show_threshold
-        return None
+        return []
```

Comment:
One can, I just tried:
```python
In [1]: import numpy as np

In [2]: np.array([])
Out[2]: array([], dtype=float64)
```

Reply:
sorry that my comment is not clear. I mean these lines
```
missing_atoms = np.zeros(self.natoms)
missing_atoms += rep.atoms_in_representaion(self.structure)
```
In case of an empty list
```
>>> import numpy as np
>>> missing_atoms = np.zeros(4)
>>> missing_atoms += []
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: operands could not be broadcast together with shapes (4,) (0,) (4,) 
>>> 
```

Reply:
Great, thanks. I see now what could be the problem. Fixed in https://github.com/aiidalab/aiidalab-widgets-base/pull/535/commits/fc815dc6619b91e0e7b30620233751f2ab85ac5d